vms/platformvm/genesis/camino.go

```diff
@@ -164,14 +176,20 @@ func (ma *MultisigAlias) Verify(txID ids.ID) error {
 		return errors.New("duplicate addresses found in multisig alias")
 	}
 
+	if ma.Threshold <= 0 || addrSet.Len() < int(ma.Threshold) {
```

Comment:
isn't threshold uint ?

Reply:
good catch - still need to check against zero